docs/src/languages/php/extensions.md

```diff
@@ -60,110 +61,111 @@ You can disable those by adding them to the `disabled_extensions` list.
-| xhprof        | *   | *   | *   |     |     |     |     |     |
-| xml           |     |     |     | *   | *   | *   | *   | *   |
-| xmlreader     |     |     |     | *   | *   | *   | *   | *   |
-| xmlrpc        | *   | *   | *   | *   | *   | *   | *   | *   |
-| xmlwriter     |     |     |     | *   | *   | *   | *   | *   |
-| xsl           | *   | *   | *   | *   | *   | *   | *   | *   |
-| yaml          |     |     |     |     | *   | *   | *   | *   |
-| zbarcode      |     |     |     | *   | *   | *   | *   |     |
-| zendopcache   | *   |     |     |     |     |     |     |     |
-| zip           |     |     |     | *   | *   | *   | *   | *   |
+| Extension        | 5.4 | 5.5 | 5.6 | 7.0 | 7.1 | 7.2 | 7.3 | 7.4 | 8.0 |
+|------------------|-----|-----|-----|-----|-----|-----|-----|-----|-----|
+| amqp             |     |     |     | *   | *   | *   | *   | *   |     |
+| apc              | *   | *   |     |     |     |     |     |     |     |
+| apcu             | *   |     | *   | *   | *   | *   | *   | *   | *   |
+| apcu_bc          |     |     |     | *   | *   | *   | *   | *   |     |
+| applepay         |     |     |     | *   | *   |     |     | *   |     |
+| bcmath           |     |     |     | *   | *   | *   | *   | *   | *   |
+| blackfire        | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| bz2              |     |     |     | *   | *   | *   | *   | *   | *   |
+| calendar         |     |     |     | *   | *   | *   | *   | *   | *   |
+| ctype            |     |     |     | *   | *   | *   | *   | *   | *   |
+| curl             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| dba              |     |     |     | *   | *   | *   | *   | *   | *   |
+| dom              |     |     |     | *   | *   | *   | *   | *   | *   |
+| enchant          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| event            |     |     |     |     | *   | *   | *   | *   |     |
+| exif             |     |     |     | *   | *   | *   | *   | *   | *   |
+| ffi              |     |     |     |     |     |     |     | *   | *   |
+| fileinfo         |     |     |     | *   | *   | *   | *   | *   | *   |
+| ftp              |     |     |     | *   | *   | *   | *   | *   | *   |
+| gd               | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| gearman          | *   | *   | *   |     |     |     |     |     |     |
+| geoip            | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| gettext          |     |     |     | *   | *   | *   | *   | *   | *   |
+| gmp              | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| http             | *   | *   |     |     |     |     |     | *   |     |
+| iconv            |     |     |     | *   | *   | *   | *   | *   | *   |
+| igbinary         |     |     |     | *   | *   | *   | *   | *   | *   |
+| imagick          | *   | *   | *   | *   | *   | *   | *   | *   |     |
```

Comment:
What's blocking this one?

Reply:
Pfiou lots of issues we have patches to make it work with older php and last time i've checked was not ready for php8